primitives/core/src/assertion.rs

```diff
@@ -74,12 +74,19 @@ impl Assertion {
 	}
 }
 
-pub const ASSERTION_FROM_DATE: [&str; 7] = [
+pub const ASSERTION_FROM_DATE: [&str; 14] = [
```

Comment:
Isn't that gonna be a _lot_ of requests? :sweat:

Reply:
No. Refer to our offline discussion.